lib/ui/compositing.dart

```diff
@@ -712,18 +721,27 @@ class SceneBuilder extends NativeFieldWrapperClass1 {
   /// Adds a [Picture] to the scene.
   ///
   /// The picture is rasterized at the given offset.
-  void addPicture(
+  ///
+  /// {@macro dart.ui.sceneBuilder.oldLayer}
+  ///
+  /// {@macro dart.ui.sceneBuilder.oldLayerVsRetained}
+  PictureEngineLayer addPicture(
```

Comment:
Since `PictureEngineLayer` is returned here and can't be null, the related logic in `web_ui` must also be implemented in this PR. And I'm not familiar with `web_ui`, so I'm not sure if my implementation is correct.

In addition, when this PR prepare to land, we need to temporarily disable some tests in `packages/flutter/test/rendering/layers_test.dart`, Because the `FakeSceneBuilder.addPicture` returns null
c.f. 
https://github.com/flutter/flutter/blob/94d7f5e6724d3e0c627a9ceb27e294c5c12ddebb/packages/flutter/test/rendering/layers_test.dart#L751-L781

Maybe we can consider changing the return value here to `PictureEngineLayer?`, so that we don't have to change the implementation of `web_ui`, and we don't have to disable `layers_test.dart`